scripts/run-front-playwright-container.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

We use arrays now, which are not a feature of (all) POSIX shells.

Reply:
Done thanks